pxr/imaging/hdx/presentTask.h

```diff
@@ -82,6 +55,12 @@ class HdxPresentTask : public HdxTask
     HDX_API
     void Execute(HdTaskContext* ctx) override;
 
+    /// Returns true if the format is supported for presentation. This is useful
+    /// for upstream tasks to prepare the AOV data accordingly, and keeps the
+    /// interop step simple.
```

Comment:
interop is jargonny, since we're eliminating the use of HgInterop here, I think we should say

```
/// ... and keeps
/// api-interoperable presentation simple.
```